samples/boards/cc3220sf_launchxl/simplelink_shell/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _simplelink_shell:
+
+SimpleLink WLAN Shell for CC3220SF WiFi
+########################################
+
+Overview
+********
+
+This sample enables testing of the WiFi offload management operations
+of the SimpleLink WiFi driver, and shows how to request and get
+notifications of WLAN events using the Zephyr network management
+framework, :file:`doc/subsystems/networking/network-management-api.rst`.
+
+Building and Running
+********************
+
+This sample implements a simple shell, prompting input from the console.
+It can be built and executed on the CC3220SF_LAUNCHXL as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/boards/cc3220sf_launchxl/simplelink_shell
+   :board: cc3220sf_launchxl
+   :goals: build
+   :compact:
+
+Sample Console Interaction
+==========================
+
+.. code-block:: console
+
+	cc3220sf wifi> scan
```

Comment:
This will display better if you indent the body of the code-block to the "c" in "code-block"